content/docs/concepts/data_model.md

```diff
@@ -17,42 +17,39 @@ Every time series is uniquely identified by its metric name and optional key-val
-   
-Note: The colons are reserved for user defined recording rules. They should not be used by exporters or direct instrumentation.
-
+* Metric names SHOULD specify the general feature of a system that is measured (e.g. `http_requests_total` - the total number of HTTP requests received). 
+* Metric names MAY use any UTF-8 characters.
+* Metric names SHOULD match the regex `[a-zA-Z_:][a-zA-Z0-9_:]*` for the best experience and compatibility (see the warning below). Metric names outside of that set will require escaping e.g. when used in PromQL (see the [UTF-8 guide](../guides/utf8.md#querying)).
 
+NOTE: The colons are reserved for user defined recording rules. They should not be used by exporters or direct instrumentation.
 
 ***Metric labels:***
 
- * Enable Prometheus's dimensional data model to identify any given combination of labels for the same metric name. It identifies a particular dimensional instantiation of that metric (for example: all HTTP requests that used the method `POST` to the `/api/tracks` handler). The query language allows filtering and aggregation based on these dimensions. 
- * The change of any label's value, including adding or removing labels, will create a new time series.
- * Labels may contain ASCII letters, numbers, as well as underscores. They must match the regex `[a-zA-Z_][a-zA-Z0-9_]*`. 
- * Label names beginning with `__` (two "_") are reserved for internal use.
- * Label values may contain any Unicode characters.
- * Labels with an empty label value are considered equivalent to labels that do not exist.
+Labels enable Prometheus's dimensional data model to identify any given combination of labels for the same metric name. It identifies a particular dimensional instantiation of that metric (for example: all HTTP requests that used the method `POST` to the `/api/tracks` handler). The query language allows filtering and aggregation based on these dimensions. The change of any label's value, including adding or removing labels, will create a new time series.
+
+* Label names MAY use any UTF-8 characters. 
+* Label names beginning with `__` (two "_") MUST be reserved for internal Prometheus use.
+* Label names SHOULD match the regex `[a-zA-Z_][a-zA-Z0-9_]*` for the best experience and compatibility (see the warning below). Label names outside of that regex will require escaping e.g. when used in PromQL (see the [UTF-8 guide](../guides/utf8.md#querying)).
+* Label values MAY contain any UTF-8 characters.
+* Labels with an empty label value are considered equivalent to labels that do not exist.
 
+WARNING: The [UTF-8](../guides/utf8.md) support for metric and label names was added relatively recently in Prometheus v3.0.0. It might take time for the wider ecosystem (downstream PromQL compatible projects and vendors, tooling, 3P instrumentation, collectors, etc.) to adopt new escaping mechanisms, relaxed validation etc. For the best compatibility it's recommended to stick to the recommended ("SHOULD") character set.
```

Comment:
3P → third-party

(Assuming that's what you mean…)

Reply:
sorry, Google jargon 👍🏽